src/utils/users.js

```diff
@@ -0,0 +1,4 @@
+export const signOut = () => {
+    localStorage.clear();
```

Comment:
Let's not clear the entire localstorage, but only the attributes we need. To do this, it is better to create a function in `utils/localstorage`.